game-core/src/main/java/games/strategy/triplea/delegate/battle/AbstractBattle.java

```diff
@@ -57,7 +57,6 @@
 
   List<Unit> attackingUnits = new ArrayList<>();
   List<Unit> defendingUnits = new ArrayList<>();
-  List<Unit> amphibiousLandAttackers = new ArrayList<>();
```

Comment:
This is a suspect I believe for save game compatibility issues. Have you double checked that this field is not serialized?

Reply:
Oddly enough, it didn't prevent loading an old save game.  But I'm adding them back in.